src/main/java/com/rapid7/container/analyzer/docker/model/LayerPathWrapper.java

```diff
@@ -0,0 +1,21 @@
+package com.rapid7.container.analyzer.docker.model;
+
+import com.rapid7.container.analyzer.docker.model.image.Layer;
+
+public class LayerPathWrapper {
+  private final Layer layer;
+  private final String basePath;
+
+  public LayerPathWrapper(String basePath, Layer layer) {
```

Comment:
just wondering if we can come up with a better name for this. Maybe just `LayerPath`?